cfme/v2v/migrations.py

```diff
@@ -293,6 +313,58 @@ class AddMigrationPlanView(View):
     next_btn = Button('Next')
     cancel_btn = Button('Cancel')
 
+    @View.nested
+    class general(View):  # noqa
+        infra_map = BootstrapSelect('infrastructure_mapping')
+        name = TextInput(name='name')
+        description = TextInput(name='description')
+        select_vm = RadioGroup('.//div[contains(@id,"vm_choice_radio")]')
+
+    @View.nested
+    class vms(View):  # noqa
+        import_btn = Button('Import')
+        importcsv = Button('Import CSV')
+        hidden_field = HiddenFileInput(locator='.//input[contains(@accept,".csv")]')
+        table = Table('.//div[contains(@class, "container-fluid")]/table',
+                      column_widgets={"Select": Checkbox(locator=".//input")})
+        filter_by_dropdown = SelectorDropdown('id', 'filterFieldTypeMenu')
+        search_box = TextInput(locator=".//div[contains(@class,'input-group')]/input")
+        clear_filters = Text(".//a[text()='Clear All Filters']")
+
+        def filter_by_name(self, vm_name):
+            try:
+                self.filter_by_dropdown.item_select("VM Name")
+            except NoSuchElementException:
+                self.logger.info("`VM Name` not present in filter dropdown!")
+            self.search_box.fill(vm_name)
+            self.browser.send_keys(Keys.ENTER, self.search_box)
+
+        def filter_by_source_cluster(self, vm_name):
```

Comment:
I think this filter and the path filter are confusing when they take a vm_name, they should call their arguments 'cluster_name' and 'path' or something of the sort.